packages/paste-website/src/pages/components/paragraph/index.mdx

```diff
@@ -0,0 +1,156 @@
+
+### Accessibility
+
+Paragraph text should be legible, and easy to read. It should also provide a clear amount of space beneath a block of text to visually separate itself from adjacent content.
+
+- Breaking content up into paragraphs helps screen readers and assistive technology provide shortcuts for users to skip between them
+- Paragraph text should meet AA requirements (4.5:1) for color contrast from itself and the background color
+
+## Examples
+
+### Default Paragraph
+
+<LivePreview scope={{Paragraph}} language="jsx">
+  {`<article>
+  <Paragraph>
+    Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <em>Curabitur blandit tempus</em> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+  </Paragraph>
+  <Paragraph>
+    Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. <strong>Curabitur blandit tempus</strong> porttitor. Nulla vitae elit libero, a pharetra augue. Donec id elit non mi porta gravida at eget metus. Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit. Nulla vitae elit libero, a pharetra augue.
+  </Paragraph>
+</article>`}
+</LivePreview>
+
+## Composition Notes
+
+### When to Use a Paragraph
+
+Paragraphs should be used for most text blocks. Paragraphs provide defaults for font-size, line-height, and margin-bottom. Paragraphs can be used within other components as well. Typically, these are anywhere from 1-5 sentences. Use paragraphs like you would when authoring a word document, to group sentences of a similar theme.
+
+<DoDont>
+  <Do title="Do" body="Use paragraphs for text blocks." center />
+  <Dont title="Don't" body="Don’t use paragraphs for spacing or when more semantic elements can be used in their place, such as headings, lists, or labels." />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Break content up into small paragraphs that allow users to skim the information." center />
+  <Dont title="Don't" body="Don’t create overly long paragraphs that make it difficult for the reader to find information." />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use paragraphs as sibling elements, with only inline elements such as bold or italic as children." center />
+  <Dont title="Don't" body="Don’t nest paragraphs within each other either, only inline elements should be nested within a paragraph." />
+</DoDont>
+
+## Anatomy
+
+| Property      | Default token      | Modifiable? |
+| ------------- | ------------------ | ----------- |
+| font-size     | font-size-30       | No          |
+| line-height   | line-height-30     | No          |
+| font-weight   | font-weight-normal | No          |
+| margin-bottom | space-60           | No          |
+| color         | color-text         | No          |
+
+---
+
+## Usage Guide
+
+### API
```

Comment:
nit: is this header even necessary?

Reply:
Yeah, it's an odd one, but it's on all the other pages, so consistency? 🤷‍♂